CONTRIBUTING.md

```diff
@@ -104,6 +104,27 @@ For discussion around this architecture see
 where it was proposed.
 
 
+### Drivers
+
+_Drivers_ implement third-party specific functionality in a standardized way.
+For example, the [GitHub driver](./src/drivers/code_hosting/github.sh)
+implements GitHub-related operations like creating a pull request there.
+
+There is also a similar
+[Bitbucket driver](./src/drivers/code_hosting/bitbucket.sh)
+that does the same things on Bitbucket.
+Both drivers are part of the _driver family_ for
+[code hosting](./src/drivers/code_hosting).
+
+The operations that a driver needs to support in order to be usable by Git Town
+are described in the documentation for the respective driver family.
+
+In order to use a driver family, all that a script needs to do is activate it
```

Comment:
Changed to 

> In order to use a driver, a script simply needs to activate the respective
driver family.
The driver family's activation script then automatically determines
the appropriate driver for the current environment and loads it.

What do you think?